### PR TITLE
feat: add new `org:ci` scope

### PR DESCRIPTION
**Pull request by mydea**
This adds a new `org:ci` scope, which allows to create & read project releases (for now). We may/will extend the rights of this scope in the future. For now, it is hidden in the UI unless an org has the `org-auth-tokens` feature flag.

This will be used for the first iteration of the org-bound access tokens.

This adds a migration:

```sql
--
-- Alter field scopes on apiauthorization
--
--
-- Alter field scopes on apikey
--
--
-- Alter field scopes on apitoken
--
--
-- Alter field scopes on sentryapp
--
```

to update the scopes field.

### CODE REVIEW

**Review by iambriccardo — APPROVED** [review]

File: src/sentry/models/apikey.py [hunk 1]
```diff
@@ -44,6 +44,7 @@ class ApiKey(Model):
             ("org:read", "org:read"),
             ("org:write", "org:write"),
             ("org:admin", "org:admin"),
+            ("org:ci", "org:ci"),
```
Comment (iambriccardo) [inline src/sentry/models/apikey.py:47]: Did you check with @mitsuhiko to use the `ApiToken` entity?

Comment (iambriccardo) [reply src/sentry/models/apikey.py:47]: How do we want to deal with the token's length?

Comment (mydea) [reply src/sentry/models/apikey.py:47]: This is unrelated to this, just adds the new scope generally! This does not add a new token/other stuff yet :)

Comment (iambriccardo) [reply src/sentry/models/apikey.py:47]: But adding a new scope would imply that other tokens can also use it, do we want this?

Comment (mydea) [reply src/sentry/models/apikey.py:47]: Yeah, AFAIK we should use the same scopes for both/all types of tokens. We may decide to simply not allow to select this for non-org-based tokens eventually, but IMHO there is no (technical) way it should be forbidden - all access that this scope has can also be achieved via other scopes :)

**Comment by mydea** [comment]
Added a DB migration, which was needed for this cc @iambriccardo

**Review by markstory — APPROVED** [review]
Migration looks good.

**Comment by mydea** [comment]
So I did some more digging together with @iambriccardo, and it _seems_ we are not actually using the BitField `scope` field anymore, but are using the `scope_list` everywhere. Which is why everything appeared to work fine even with the bitfield adjusted in the middle. But I guess it is technically incorrect, so should still be added at the end of the bitfield list?? I adjusted it this way now, but we should get a definite answer that this is correct this way (and maybe think about removing the bitfield `scope` if we don't need it anymore? but that's a completely different task then :D )

**Pull request closed without merging**